packages/oc-s3-storage-adapter/index.js

```diff
@@ -27,25 +27,52 @@ module.exports = function(conf) {
     return true;
   };
 
+  // Defaults
+  const bucket = conf.bucket ? conf.bucket : '';
+  const sslEnabled = conf.sslEnabled === false ? false : true;
+  const s3ForcePathStyle = conf.s3ForcePathStyle ? true : false;
+  const signatureVersion = conf.signatureVersion
+    ? conf.signatureVersion
+    : AWS.Config.signatureVersion;
   const httpOptions = { timeout: conf.timeout || 10000 };
   if (conf.agentProxy) {
     httpOptions.agent = conf.agentProxy;
   }
 
-  AWS.config.update({
+  // Setup AWS config
+  let awsConfig = new AWS.Config({
     accessKeyId: conf.key,
     secretAccessKey: conf.secret,
-    region: conf.region,
```

Comment:
With the removal of region, won't this break existing users using the AWS s3?

Reply:
Yes, you're right, let's add that back. All user's are not likely to use the default us-east-1.